android/assets/jsons/Civ V - Vanilla/Nations.json

```diff
@@ -208,8 +208,8 @@
 		"startBias":["Avoid [Tundra]","Avoid [Desert]"],
 		"preferredVictoryType": "Scientific",
 
-        "startIntroPart1": "",
-        "startIntroPart2": "",
+        "startIntroPart1": "The blessing of the gods be upon you, Caesar Augustus, emperor of Rome and all her holdings. Your empire was the greatest and longest lived of all in Western civilization. And your people single handedly shaped its culture, law, art, and warfare like none other, before or since. Through years of glorious conquest, Rome came to dominate all the lands of the Mediterranean from Spain in the West to Syria in the east. And her dominion would eventually expand to cover much of England and northern Germany. Roman art and architecture still awe and inspire the world. And she remains the envy of all lesser civilizations who have followed.",
+        "startIntroPart2": "O mighty emperor, your people turn to you to once more reclaim the glory of Rome! Will you see to it that your empire rises again, bringing peace and order to all? Will you make Rome once again center of the world? Can you build a civilization that will stand the test of time?",
```

Comment:
Is this your text?

Reply:
That is to say - did you write this from scratch or is it copied from somewhere?

Reply:
It's exactly the same lines than in Civ5. :sweat_smile: 
<details><summary>From civ5 game file</summary>
DawnOfManQuote=[=[The blessings of the gods be upon you, Caesar Augustus, Emperor of Rome and all her holdings. Your empire was the greatest and longest-lived of all in Western Civilization, and your people single-handedly have shaped its culture, law, art, and warfare like none other, before or since. Through years of glorious conquest, Rome came to dominate all the lands of the Mediterranean, from Spain in the west to Syria in the east, and her dominion would eventually expand to cover much of England and Northern Germany. Roman art and architecture still awe and inspire the world, and she remains the envy of all of the lesser civilizations who have followed.

O mighty Emperor, your people turn to you to once more reclaim the glory of Rome. Will you see to it that your empire rises again, bringing peace and order to all?  Will you make Rome once again center of the world? Can you build a civilization that will stand the test of time?]=];
DawnOfManImage="DOM_Augustus.dds";
DawnOfManAudio="AS2D_DOM_SPEECH_ROME";};
</details>

Reply:
In that case, denied. These were removed for copyright reasons